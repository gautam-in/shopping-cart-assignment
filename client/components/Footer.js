import { FooterStyle } from "../styles/FooterStyle";

const Footer = () => {
  return (
    <FooterStyle>
      <div>
        Copyright &copy; 2011-2018 Sabka Bazaar Grocery Supplies Pvt Ltd
      </div>
    </FooterStyle>
  );
};

export default Footer;
