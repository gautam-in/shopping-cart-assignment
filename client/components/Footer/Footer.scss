$light-gray:#c8cbce61;

.footer {
    height: 2rem;
    display: flex;
    align-items: center;
    justify-content: center;
    background: $light-gray;
    width: 100%;
    font-size: 10px;
}