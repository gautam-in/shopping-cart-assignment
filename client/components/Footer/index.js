import { footerText } from './footer.module.scss'

const Footer = () => {
    return (
        <footer>
            <p className={footerText}>Copyright &copy; 2011 - 2022</p>{' '}
            <p className={footerText}>Sabka Bazaar Grocery Supplies Pvt Ltd</p>
        </footer>
    )
}


export default Footer