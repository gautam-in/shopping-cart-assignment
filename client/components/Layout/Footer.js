import styles from '../../styles/footer.module.scss';

const Footer = () => {
    return (
        <footer className={styles.footer}>
            <p style={{ padding: "10px", margin: "0" }}>Copyright Â© 2011-2018 Sabka Bazaar Grocery Supplies Pvt Ltd </p>
        </footer>
    )
}

export default Footer
