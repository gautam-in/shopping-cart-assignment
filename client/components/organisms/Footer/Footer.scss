/* @import "/client/styles/variables"; */
@import "../../../styles/variables";
@import "/client/styles/mq-mixin";

footer {
  color: $primary-black;
  background-color: $bg-secondary-color;
  font-size: $primary-font-size;
  font-weight: 600;
  padding: 10px;
  position: fixed;
  bottom: 0px;
  width: 100%;
  /* margin-top: 20px; */
  text-align: center;

  @include mq(desktop) {
    text-align: left;
    padding-left: 0px;
  }
}
