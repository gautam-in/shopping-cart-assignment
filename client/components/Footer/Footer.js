import React from "react";
import "./Footer.scss";

const Footer = () => {
  return (
    <div className="footer">
      copyright &copy; 2021-2022 Sabka bazaar grocery supplies Pvt. Ltd.
    </div>
  );
};

export default Footer;
