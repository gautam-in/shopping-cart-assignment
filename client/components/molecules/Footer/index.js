import { FooterWrapper } from "./Footer.styles";

const Footer = () => (
  <FooterWrapper>
    Copyright &#169; 2011-2021 Sabka Bazaar Grocery Supplies Pvt Ltd.
  </FooterWrapper>
);

export default Footer;
