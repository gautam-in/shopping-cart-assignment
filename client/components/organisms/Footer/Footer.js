import React from "react";
import "./Footer.scss";

function Footer() {
  return (
    <footer data-test="component-footer">
      <p className="parent_spacing">
        Copyright @ 2011-2018 Sabka Bazaar Grocery Supplies Pvt Ltd
      </p>
    </footer>
  );
}

export default Footer;
