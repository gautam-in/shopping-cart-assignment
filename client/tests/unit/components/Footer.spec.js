import { mount, createLocalVue } from "@vue/test-utils";
import Footer from "@/components/Footer.vue";

const localVue = createLocalVue();
localVue.use({
  Footer,
});

describe("Footer", () => {
  it("snapshot", () => {
    const wrapper = mount(Footer, {
      localVue,
    });
    expect(wrapper).toMatchSnapshot();
    // expect(wrapper.contains("div")).toBe(true);
  });
});
