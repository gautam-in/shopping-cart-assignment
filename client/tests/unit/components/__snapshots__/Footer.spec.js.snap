// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer snapshot 1`] = `<div>Copyright Â© 2011-2018 Sabka Bazaar Grocery Supplies Pvt Ltd</div>`;
